icc-x-api/icc-bekmehr-x-api.ts

```diff
@@ -111,7 +111,7 @@ export class IccBekmehrXApi extends iccBekmehrApi {
   ): Promise<Blob> {
     return new Promise((resolve, reject) => {
       const socket = new WebSocket(
-        `${this.wssHost}/be_kmehr/generateSmf${sessionId ? `;jsessionid=${sessionId}` : ""}`
+        `${this.wssHost}/be_kmehr/generateSumehr${sessionId ? `;jsessionid=${sessionId}` : ""}`
```

Comment:
@olmartin @phil1618 What do you think?

Reply:
I wonder why not using `generateSumehrV2` but anyway we do not use `generateSumehrExportWithEncryptionSupport` in Medispring so it would change nothing for us.